pkgs/tools/misc/pmbootstrap/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, git, openssl, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "pmbootstrap";
+  version = "1.30.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-uHN3nplQOMuBeQIxAocCVqwnmJUQZL67+iXLhQ7onps=";
+  };
+
+  buildInputs = [ ];
+
+  propagatedBuildInputs = [ git openssl ];
+
+  doCheck = false;
```

Comment:
Why did you disable tests?

Reply:
I would need some help here. I enabled tests with pytest now, but it fails as all tests require a pmb_test module included within the project git repo. I am not sure how to go about it. I tried going through some other python modules using pytest and buildPythonApplication but haven't found anything like this yet.

Reply:
Maybe you need to the directory to PYTHON_PATH.

Reply:
I added directory to PYTHON_PATH, but many tests require sudo and network access. We may have to disable them after all. Please refer to 'experiment' commit in latest revision to verify if possible.